### PR TITLE
943 Biography Brick

### PR DESCRIPTION
**Pull request by robbdavis**
943 Biography Brick

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  <div class="row">
+    <div class="container">
+      <div class="row">
+
+        <div class="col-sm-12">
+          <div class="row header <?php print $alignment; ?>">
+            <div class="col-sm-8 col-sm-offset-2">
+
+                <?php if ($show_heading == 0){ ?>
+                <h3><?php print $main_heading; ?></h3>
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php]: Indent this line.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+                <?php if (isset ($main_description)){ ?>
+                    <p><?php print $main_description; ?></p>
+                <?php }; ?>
+
+            </div>
+          </div>
+        </div>
+
+        <?php   foreach ($content['field_gbl_bio_biography']['#items'] as $item) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php]: Extra spacing.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                // Otherwise set the variables to empty so they don't throw errors when they are called
+                $bio_img = $alt = '';
+            }
+
+            if (isset($loaded_entity->field_fc_bio_description)) {
+                $bio_description = $loaded_entity->field_fc_bio_description[LANGUAGE_NONE][0]['value'];
+            }
+
+        ?>
+
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php]: Add some comments for this final column and what is happening here.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/ucgy_gbl_biography.module [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        $brick_classes = array('biography');
+
+        // Merge all the classes together into a happy bundle
+        $vars['classes_array'] = array_merge($vars['classes_array'], $brick_classes);
+    }
+}
+
+/**
+ * Implements hook_form_alter().
+ */
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/ucgy_gbl_biography.module]: Should add a general comment here for the purpose of the hook_form_alter.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php (hunk 1 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php]: fixed

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php (hunk 2 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php]: fixed

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php (hunk 3 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/templates/fieldable-panels-pane--gbl_biography.tpl.php]: fixed

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/ucgy_gbl_biography.module (hunk 4 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_biography/ucgy_gbl_biography.module]: added

**Comment by robbdavis** [comment]
Should be good now dan.

**Pull request merged**